### PR TITLE
NXP-23752: fix form validation

### PR DESCRIPTION
**Pull request by Gabez0r**

### CODE REVIEW

File: elements/directory/nuxeo-vocabulary-management.html [hunk 1]
```diff
@@ -318,33 +318,34 @@ <h2>[[i18n('vocabularyManagement.popup.editEntry')]]</h2>
...
-            this.$.vocabularyEditDialog.toggle();
-            this.fire('notify', {message: this.i18n('vocabularyManagement.successfullyEdited')});
-            this._refresh();
-          }.bind(this),
-          function(err) {
-            this.fire('notify', {message: this.i18n('label.error').toUpperCase() + ': ' +
-                                          (err.message && err.message.length > 0 ? err.message :
-                                           this.i18n('vocabularyManagement.cannotEdit'))});
-          }.bind(this));
+        if (this.$.layout.validate()) {
```
Comment (afjusto) [inline elements/directory/nuxeo-vocabulary-management.html]: just return early if condition fails instead of changing all of this?

**Comment by afjusto** [comment]
👍

File: elements/directory/nuxeo-vocabulary-management.html (hunk 1 above)
Comment (Gabez0r) [reply elements/directory/nuxeo-vocabulary-management.html]: 👍

Comment (Gabez0r) [reply elements/directory/nuxeo-vocabulary-management.html]: 👍

**Comment by nelsonsilva** [comment]
:+1:

**Comment by guirenard** [comment]
:+1:

**Pull request merged**